include/VecCore/VecMath.h

```diff
@@ -80,23 +64,116 @@ T Sign(const T &x)
   return CopySign(T(1), x);
 }
 
-// Trigonometric Functions
+#ifdef VECCORE_ENABLE_LIBMVEC
+
+#define VECMATH_REAL_FUNC2(f, name)                          \
+  template <typename T>                                      \
+  VECCORE_FORCE_INLINE                                       \
+  VECCORE_ATT_HOST_DEVICE                                    \
+  T f(const T &x, const T &y)                                \
+  {                                                          \
+      T output;                                              \
+      int v_size = vecCore::VectorSize(x);                   \
+                                                             \
+      #pragma omp simd                                       \
+      for(int i = 0; i < v_size; i++)                        \
+      {                                                      \
+        Scalar<T> input1 = vecCore::Get(x, i)                \
+        Scalar<T> input2 = vecCore::Get(y, i)                \
+        vecCore::Set(output, i, std::name(input1, input2));  \
+      }                                                      \
```

Comment:
This code is inside the `vecCore` namespace. You could write `Set(output, i, std::name(Get(x, i), Get(y, i))`, no?

Reply:
I think I changed it locally, but forgot to push..